client/performance-profiler/components/charts/history-chart.tsx

```diff
@@ -0,0 +1,211 @@
+		.range( [ margin.left + 20, width - margin.right - 20 ] );
+
+	const yScale = d3ScaleLinear()
+		.domain( [ 0, d3Max( data, ( item: chartItem ) => item.value ) ] )
+		.nice()
+		.range( [ height - margin.bottom, margin.top ] );
+
+	const colorScale = d3ScaleThreshold()
+		.domain( range )
+		.range( [ '#00BA37', '#D67709', '#D63638' ] );
+
+	return { xScale, yScale, colorScale };
+};
+
+// Initialize SVG and set dimensions
+const initializeSVG = ( svgRef, width, height ) => {
+	return d3Select( svgRef.current ).attr( 'width', width ).attr( 'height', height );
+};
+
+// Create gradient definition
+const createGradient = ( svg, data, xScale, colorScale, margin, width ) => {
+	const defs = svg.append( 'defs' );
+	const gradient = defs
+		.append( 'linearGradient' )
+		.attr( 'id', 'line-gradient' )
+		.attr( 'gradientUnits', 'userSpaceOnUse' )
+		.attr( 'x1', margin.left )
+		.attr( 'y1', 0 )
+		.attr( 'x2', width - margin.right )
+		.attr( 'y2', 0 );
+
+	data.forEach( ( item: chartItem ) => {
+		gradient
+			.append( 'stop' )
+			.attr(
+				'offset',
+				`${
+					( ( xScale( new Date( item.date ) ) - margin.left ) / ( width - margin.left ) ) * 100
+				}%`
+			)
+			.attr( 'stop-color', colorScale( item.value ) )
+			.attr( 'stop-opacity', 0.3 );
+	} );
+};
+
+// Draw grid lines
+const drawGrid = ( svg, yScale, width, margin ) => {
+	svg
+		.selectAll( 'line.horizontal-grid' )
+		.data( yScale.ticks( 6 ) )
+		.enter()
+		.append( 'line' )
+		.attr( 'class', 'horizontal-grid' )
+		.attr( 'x1', margin.left )
+		.attr( 'x2', width - margin.right )
+		.attr( 'y1', yScale )
+		.attr( 'y2', yScale )
+		.attr( 'stroke', '#F6F7F7' )
+		.attr( 'stroke-width', 2 );
+};
+
+// Draw line with gradient
+const drawLine = ( svg, data, xScale, yScale ) => {
+	const lineGenerator = d3Line()
+		.x( ( item: chartItem ) => xScale( new Date( item.date ) ) )
+		.y( ( item: chartItem ) => yScale( item.value ) )
+		.curve( d3MonotoneXCurve );
+
+	svg
+		.append( 'path' )
+		.datum( data )
+		.attr( 'd', lineGenerator )
+		.attr( 'stroke', 'url(#line-gradient)' )
+		.attr( 'stroke-width', 4 )
+		.attr( 'fill', 'none' );
+};
+
+// Draw axes
+const drawAxes = ( svg, xScale, yScale, data, margin, width, height ) => {
+	const dates = data.map( ( item: chartItem ) => new Date( item.date ) );
+
+	svg
+		.append( 'g' )
+		.attr( 'transform', `translate(0,${ height - margin.bottom })` )
+		.call(
+			d3AxisBottom( xScale )
+				.tickValues( dates )
+				.tickFormat( d3TimeFormat( '%m/%d' ) )
+				.tickPadding( 10 )
+		)
+		.call( ( g ) => g.select( '.domain' ).remove() );
+
+	svg
+		.append( 'g' )
+		.attr( 'transform', `translate(${ margin.left },0)` )
+		.call( d3AxisLeft( yScale ).ticks( 6 ) )
+		.select( '.domain' )
+		.remove();
+};
+
+// Create shape path for dots
+const createShapePath = ( item, xScale, yScale, range ) => {
+	const x = xScale( new Date( item.date ) );
+	const y = yScale( item.value );
+	const size = 7;
```

Comment:
If I understood properly this is what sets the icon to have a `14px` width, on Figma we have all of them with `12px` and they look slightly smaller. 

If this is simple as it seems, I think it worth to make the change now.

Reply:
The icons appear smaller because the graph produced in this pull request generates a larger chart. When the chart is rendered at a size of 600px, the icon size looks good. I will wait until the final designs are implemented; if the icons still look smaller, then I will update the size.